samples/javascript_nodejs/57.teams-conversation-bot/bots/teamsConversationBot.js

```diff
@@ -149,8 +144,7 @@ class TeamsConversationBot extends TeamsActivityHandler {
             await context.sendActivity(message);
         } catch (e) {
             if (e.code === 'MemberNotFoundInConversation') {
-                context.sendActivity(MessageFactory.text('Member not found.'));
-                return;
+                return context.sendActivity(MessageFactory.text('Member not found.'));
```

Comment:
Return the promise so that a consumer that awaits this function will have consistent behavior.